dolt_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Introducing Dolt + Metaflow
+---
+
+## Background
+This post details how to use Metaflow with Dolt. [Metaflow](https://metaflow.org/) is a workflow manager that offers data scientists the ability to define local experiments and scale those experiments to production jobs from a single API. [Dolt](https://docs.dolthub.com/) is a version controlled relational database. It provides a familiar SQL interface along with Git-like version control features for capturing historical states. Each commit corresponds to a complete state of the database at the time the commit was created. Both Dolt and Metaflow are open source.
+
+Using Dolt as a source and target for Metaflow jobs that users can reproduce, trace the lineage of, and employ complex back testing logic against.
+
+## Example Pipeline
+Our pipeline consists of two flows. One flow consumes the results written the other. The first flow computes the state level median price for a procedure. The second flow computes the variance of the price for a procedure across states. 
+
+![image here](dolt-metaflow-pipeline.png)
+
+First we will form and test a hypothesis using our pipeline, a bit of Arm Chair Data Science. Then we will dive into some of the more capabilities the integration with Dolt offers Metaflow users. We will show how to use the Metaflow API with Dolt integration to:
+- rerun the second flow using a pinned version of the data produced by the first flow, providing read isolation and reproducibility to our variance calculation
+- given the result of the second flow trace the lineage of the data back to the upstream dataset
+
+We are able to do all of this from the Metaflow API, allowing users to access powerful data version control features from the comfort of their workflow framework.
+
+## How it Works
+Workflows in Metaflow are called "flows." Each flow stores metadata about flow execution when it is run, referred to as a "run." Each time a run using the integration interacts with Dolt, it captures a small amount of metadata that makes that interaction completely reproducible. Since Dolt database root hashes are unique, this creates a mapping (roughly) between Metaflow runs and Dolt commits that can be exploited to provide users with powerful lineage and reproducibility features. 
+
+At the level of the Flow, the integration looks like this:
+
+![Flow level view](dolt-metaflow-integration-flow-level.png)
+
+Drilling into one of the runs:
+
+![Run level view](dolt-metaflow-integration-run-level.png)
+
+This is all abstract, so let's install a few dependencies, grab a dataset, and get stuck some armchair data science.
```

Comment:
"get stuck some arm chair data science"

Reply:
if you don't like the phrase you can drop. some way to suggest upfront that we aren't being rigorous with our analysis seems like a good idea